src/apps/bridge/sensor_drivers/rbrCodaSensor.cpp

```diff
@@ -119,14 +119,18 @@ void RbrCodaSensor::aggregate(void) {
     aggs.sensor_type = rbrCodaGetSensorType(node_id);
     if (temp_deg_c.getNumSamples() >= MIN_READINGS_FOR_AGGREGATION) {
       aggs.temp_mean_deg_c = temp_deg_c.getMean();
-      aggs.pressure_mean_deci_bar = pressure_ubar.getMean();
-      aggs.pressure_stdev_deci_bar = pressure_ubar.getStd(aggs.pressure_mean_deci_bar);
+      aggs.pressure_mean_deci_bar = pressure_deci_bar.getMean();
+      aggs.pressure_stdev_deci_bar = pressure_deci_bar.getStd(aggs.pressure_mean_deci_bar);
       aggs.reading_count = reading_count;
 
-      if (aggs.temp_mean_deg_c < TEMP_SAMPLE_MEMBER_MIN ||
-          aggs.temp_mean_deg_c > TEMP_SAMPLE_MEMBER_MAX) {
+      if (aggs.temp_mean_deg_c < TEMP_SAMPLE_MEMBER_MIN) {
+        aggs.temp_mean_deg_c = -HUGE_VAL;
+      } else if (aggs.temp_mean_deg_c > TEMP_SAMPLE_MEMBER_MAX) {
+        aggs.temp_mean_deg_c = HUGE_VAL;
+      } else {
         aggs.temp_mean_deg_c = NAN;
       }
```

Comment:
Section 1.30 of the math docs at https://sourceware.org/newlib/ says:

> `infinity` and `infinityf` are neither standard C nor POSIX. C and POSIX require macros `HUGE_VAL` and `HUGE_VALF` to be defined in `math.h`, which Newlib defines to be infinities corresponding to these archaic `infinity()` and `infinityf()` functions in floating-point implementations which do have infinities. 

This particular change encodes temperatures that are valid readings from the RBR sensor but which fall outside the range that we can meaningfully aggregate and encode as positive or negative "infinity" or `HUGE_VAL`.